src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -1152,4 +1240,440 @@
       it { expect { post :move_path, params: { project: 'non:existent:project' } }.to raise_error ActiveRecord::RecordNotFound }
     end
   end
+
+  describe 'GET #monitor' do
+    let(:repo_for_user) { create(:repository, name: 'openSUSE_Tumbleweed', project: user.home_project) }
+    let(:arch_i586) { Architecture.where(name: 'i586').first }
+    let(:arch_x86_64) { Architecture.where(name: 'x86_64').first }
+    let!(:package) { create(:package, project: user.home_project) }
+
+    context 'with a nonexistent project' do
```

Comment:
This is not testing `monitor`, this is testing `set_project` which is already tested in `spec/controllers/webui/webui_controller_spec.rb`